qupath_plus.def

```diff
@@ -0,0 +1,54 @@
+Bootstrap: docker
+From: amd64/ubuntu:jammy
```

Comment:
the amd64 was needed for test builds on arm64, but it's a good safety because the qupath linux binaries are amd64